pkg/gadgets/trace/fsslower/types/types.go

```diff
@@ -25,14 +26,18 @@ const (
 type Event struct {
 	eventtypes.Event
 
-	MountNsID uint64 `json:"mountnsid,omitempty"`
-	Comm      string `json:"comm,omitempty"`
-	Pid       uint32 `json:"pid,omitempty"`
-	Op        string `json:"op,omitempty"`
-	Bytes     uint64 `json:"bytes,omitempty"`
-	Offset    int64  `json:"offset,omitempty"`
-	Latency   uint64 `json:"latency,omitempty"`
-	File      string `json:"file,omitempty"`
+	MountNsID uint64 `json:"mountnsid,omitempty" column:"mnt_ns_id,hide"`
+	Pid       uint32 `json:"pid,omitempty" column:"pid,width:5,maxWidth:7"`
+	Comm      string `json:"comm,omitempty" column:"comm,width:10"`
+	Op        string `json:"op,omitempty" column:"T,maxWidth:3"`
+	Bytes     uint64 `json:"bytes,omitempty" column:"bytes,width:10"`
+	Offset    int64  `json:"offset,omitempty" column:"offset,width:10"`
+	Latency   uint64 `json:"latency,omitempty" column:"lat,width:10"`
+	File      string `json:"file,omitempty" column:"file,width:30"`
```

Comment:
Just a couple of remarks on the column settings:

[Here's](https://github.com/kinvolk/inspektor-gadget/blob/main/pkg/gadgets/trace/exec/types/types.go#L27) an example of names/widths/settings we're already using for some of the fields that are also present here, like `mountns` / `pid` / `comm` - I think we should reuse those here as well.

I'd also right align `bytes`, `offset` and `latency` using `align:right` for better readability. `T`/`Op` seems as if it can use a fixed width of 1 (`width:1,fixed`).

MaxLength of `file` can be set to 32 (see [here](https://github.com/kinvolk/inspektor-gadget/blob/784fa014c715fd4f1f338f4a3122f6b066b0edee/pkg/gadgets/trace/fsslower/tracer/bpf/fsslower.h#L5)).

Reply:
Updated with your suggestions.
Would it be possible to defines these values somewhere once and use it in every setting?

Reply:
Yeah, thought about that too while writing the comment. We could either embed structs or somehow give the opportunity to register defaults/templates based on the column names with the lib itself. But that sounds pretty hacky.

Reply:
Yeah, it would be too hacky with the lib itself, but I agree we should have it in a common place to avoid redefining it in every gadget.